common/src/main/java/com/gregtechceu/gtceu/common/data/GTBlocks.java

```diff
@@ -204,6 +205,31 @@ public static void generateMaterialBlocks() {
         MATERIAL_BLOCKS = builder.build();
     }
 
+    public static Map<Material, BlockEntry<SurfaceRockBlock>> SURFACE_ROCK_BLOCKS;
+    public static void generateSurfaceRocks() {
+        ImmutableMap.Builder<Material, BlockEntry<SurfaceRockBlock>> builder = ImmutableMap.builder();
+        for (Material material : GTRegistries.MATERIALS) {
+            if (material.hasProperty(PropertyKey.ORE)) {
+                var entry = REGISTRATE.block("%s_indicator".formatted(material.getName()), p -> new SurfaceRockBlock(p, material))
+                        .initialProperties(() -> Blocks.GRAVEL)
+                        .properties(p -> p.noLootTable().strength(0.25f))
+                        .blockstate(NonNullBiConsumer.noop())
+                        .setData(ProviderType.LANG, NonNullBiConsumer.noop())
+                        .setData(ProviderType.LOOT, NonNullBiConsumer.noop())
+                        .addLayer(() -> RenderType::cutoutMipped)
+                        .color(() -> SurfaceRockBlock::tintedColor)
+                        .item(SurfaceRockItem::new)
+                        .onRegister(SurfaceRockItem::onRegister)
+                        .model(NonNullBiConsumer.noop())
+                        .color(() -> SurfaceRockItem::tintColor)
+                        .build()
```

Comment:
these shouldn't have items.

Reply:
Already resolved, had forgotten to remove ^^